blosc/bitshuffle-avx2.c

```diff
@@ -19,8 +19,8 @@
 
 /* Make sure AVX2 is available for the compilation target and compiler. */
 #if !defined(__AVX2__)
-  #error AVX2 is not supported by the target architecture/platform and/or this compiler.
-#endif
+  #warning AVX2 is not supported by the target architecture/platform and/or this compiler.
```

Comment:
Don't print a warning, it is annoying to have to ignore it and it fails to compile with `-Werror`

Reply:
I removed them.